File: app/src/main/java/info/nightscout/androidaps/plugins/pump/medtronic/comm/history/MedtronicHistoryEntryInterface.java
package info.nightscout.androidaps.plugins.pump.medtronic.comm.history;

import java.util.List;

/**
 * Created by andy on 7/24/18.
 */
public interface MedtronicHistoryEntryInterface {

    String getEntryTypeName();

    void setData(List<Byte> listRawData, boolean doNotProcess);

    int getDateLength();

}

File: medtronic/src/main/java/info/nightscout/androidaps/plugins/pump/medtronic/comm/history/MedtronicHistoryEntryInterface.kt
package info.nightscout.androidaps.plugins.pump.medtronic.comm.history

/**
 * Created by andy on 7/24/18.
 */
interface MedtronicHistoryEntryInterface {

    val entryTypeName: String
    fun setData(listRawData: MutableList<Byte>, doNotProcess: Boolean)
    val dateLength: Int
}